src/workspaces/NewWorkspaceModal/NewWorkspaceModal.test.ts

```diff
@@ -87,114 +87,116 @@ describe('NewWorkspaceModal', () => {
     return availableBillingProjectOptions.map((opt) => opt.split('.svg')[1]);
   };
 
-  it('shows a message if there are no billing projects to use for creation', async () => {
-    // Arrange
-    asMockedFn(Ajax).mockImplementation(
-      () =>
-        ({
-          Billing: {
-            listProjects: async () => [],
-          },
-          ...nonBillingAjax,
-        } as AjaxContract)
-    );
-
-    // Act
-    await act(async () => {
-      render(
-        h(NewWorkspaceModal, {
-          onSuccess: () => {},
-          onDismiss: () => {},
-        })
+  describe('handles when no appropriate billing projects are available', () => {
```

Comment:
This is a huge test file, so to help with readability I added grouping of tests. I recommend you review this with whitespace changes hidden.

Test output after grouping:

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/7732069c-cefa-4b1d-84cb-c7353fd6a763)